devito/arch/archinfo.py

```diff
@@ -329,6 +329,67 @@ def cbk(deviceid=0):
     except OSError:
         pass
 
+    # *** Third try: `sycl-ls`
```

Comment:
Purely for my own understanding, what is this doing?

Reply:
akin to `nvidia-smi` or `rocm-smi` -- returns a textual representation of the underlying GPUs